R/check_params.R

```diff
@@ -90,9 +90,11 @@ check_params = function(params_sleep = c(), params_metrics = c(),
                        "boutcriter.in", "boutcriter.lig", "boutcriter.mvpa",
                        "threshold.lig", "threshold.mod", "threshold.vig", "boutdur.mvpa",
                        "boutdur.in", "boutdur.lig")
+    character_params = c("frag.metrics", "part6_threshold_combi")
     check_class("phyact", params = params_phyact, parnames = numeric_params, parclass = "numeric")
```

Comment:
Thanks,

> I would add an extra check for part6_threshold_combi...

Thanks, I am now adding this.

> And another check that

This was partly in the check_params() with a minor variation:

```
if (params_247[["part6HCA"]] == TRUE) {
  # Add RData because part 6 will need it
  params_247[["save_ms5raw_format"]] = unique(c(params_247[["save_ms5raw_format"]], "RData"))
  params_247[["save_ms5rawlevels"]] = TRUE
  params_output[["save_ms5raw_without_invalid"]] = FALSE
}
```
In this way the user can still ask for .csv files if they want them.